doc/cephfs/charmap.rst

```diff
@@ -0,0 +1,185 @@
+CephFS Directory Entry Name Normalization and Case Folding
+==========================================================
+
+CephFS allows configuring directory trees to **normalize** or **case fold**
```

Comment:
Does _normalize_ here mean the same as _case fold_?  If so I suggest 

>to _normalize_ (aka _case fold_)

I spent a while trying to find "case fold" in lines 13-15, thinking it was slightly different, like Straße to strasse vs straße.  If they are distinct, is that distinction between the Unicode normalization forms mentioned below?

Alternately on line 14 

>of Unicode normalization or case folding to use

if that makes more sense.  There seem to be subtleties here and I seek for us to be clear.

Reply:
> Does _normalize_ here mean the same as _case fold_?

No, they are different.

I've tried to improve the wording, PTAL.

Reply:
Added `possibly`?  lgtm, that improves ambiguity.